### PR TITLE
[APM-587] API Specification: static content published manually

### PR DESCRIPTION
**Pull request by ziemsky**
Custom API Specification document + basic setup for publishing its HTML content manually.

### CODE REVIEW

File: repository-data/development/src/main/resources/hcm-config/main.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@
 definitions:
   config:
     /hippo:configuration/hippo:modules/autoexport/hippo:moduleconfig:
-      autoexport:enabled: false
```
Comment (GeoffHayward) [inline repository-data/development/src/main/resources/hcm-config/main.yaml:5]: This change will impact the teams established workflow. It's therefore better not to commit this.

File: repository-data/webfiles/src/main/resources/site/freemarker/common/apispecification.ftl [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+        </div>
+
+
+<#--    <#if document??>-->
+<#--    <code>-->
+<#--    </code>-->
+<#--    <#else>-->
+<#--        Actual document not available-->
+<#--    </#if>-->
```
Comment (GeoffHayward) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/apispecification.ftl:17]: Please don't commit commended out code.

File: repository-data/development/src/main/resources/hcm-config/main.yaml (hunk 1 above)
Comment (ziemsky) [reply repository-data/development/src/main/resources/hcm-config/main.yaml:5]: Good spot - I missed that!

File: repository-data/webfiles/src/main/resources/site/freemarker/common/apispecification.ftl (hunk 2 above)
Comment (ziemsky) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/apispecification.ftl:17]: Removing now :+1:

Comment (ziemsky) [reply repository-data/webfiles/src/main/resources/site/freemarker/common/apispecification.ftl]: Addressed in #45667b4

File: repository-data/development/src/main/resources/hcm-config/main.yaml (hunk 1 above)
Comment (ziemsky) [reply repository-data/development/src/main/resources/hcm-config/main.yaml]: Addressed in #45667b4

**Comment by ziemsky** [comment]
@GeoffHayward has approved this Pull Request but merging is blocked by results from Travis.

Travis doesn't respond, seemingly because the build isn't currently configured for Travis to support Pull Requests against branches other than `master`.

I'm closing this Pull Request and merging its changes into the long running feature branch, with integration with Travis to be addressed later, possible through the use of 'draft' Pull Requests from long running feature branch against `master`.

**Pull request closed without merging**